sys/linux/init_test.go

```diff
@@ -130,6 +130,19 @@ syz_open_dev$tty1(0xc, 0x4, 0x5)
 syz_open_dev$tty1(0xc, 0x4, 0x4)
 syz_open_dev$tty1(0xc, 0x4, 0x4)
 syz_open_dev$tty1(0xc, 0x4, 0x1)
+`,
+		},
+		{
+			In: `syz_open_dev$MSR(0x0, 0x0, 0x0)`,
+		},
+		{
+			In: `
+ioctl$X86_IOC_RDMSR_REGS(0xa, 0xc02063a0, 0x0)
+ioctl$X86_IOC_WRMSR_REGS(0xa, 0xc02063a1, 0x0)
```

Comment:
now, `make test` is not happy as is cannot find the definition of `ioctl$X86_IOC_WRMSR_REGS`

Reply:
I think you can use X86_IOC_RDMSR_REGS in both cases:
```
ioctl$X86_IOC_RDMSR_REGS(0xa, 0xc02063a0, 0x0)
ioctl$X86_IOC_RDMSR_REGS(0xa, 0xc02063a1, 0x0)
```